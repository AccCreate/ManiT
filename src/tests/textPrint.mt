a = " _______ _________ _______  _______           _______  _          _______  ______            _______  _______  ______   _______";
b = "(  ____ \\\\\\\\__   __/(  ____ \\\\(  ____ )|\\\\     /|(  ____ \\\\( (    /|  (  ____ \\\\(  __  \\\\ |\\\\     /|(  ___  )(  ____ )(  __  \\\\ (  ____ \\\\";
c = "| (    \\\\/   ) (   | (    \\\\/| (    )|| )   ( || (    \\\\/|  \\\\  ( |  | (    \\\\/| (  \\\\  )| )   ( || (   ) || (    )|| (  \\\\  )| (    \\\\/";
d = "| (_____    | |   | (__    | (____)|| (___) || (__    |   \\\\ | |  | (__    | |   ) || | _ | || (___) || (____)|| |   ) || (_____";
e = "(_____  )   | |   |  __)   |  _____)|  ___  ||  __)   | (\\\\ \\\\) |  |  __)   | |   | || |( )| ||  ___  ||     __)| |   | |(_____  )";
f = "      ) |   | |   | (      | (      | (   ) || (      | | \\\\   |  | (      | |   ) || || || || (   ) || (\\\\ (   | |   ) |      ) |";
g = "/\\\\____) |   | |   | (____/\\\\| )      | )   ( || (____/\\\\| )  \\\\  |  | (____/\\\\| (__/  )| () () || )   ( || ) \\\\ \\\\__| (__/  )/\\\\____) |";
h = "\\\\_______)   )_(   (_______/|/       |/     \\\\|(_______/|/    )_)  (_______/(______/ (_______)|/     \\\\||/   \\\\__/(______/ \\\\_______)";

print(a);
print(b);
print(c);
print(d);
print(e);
print(f);
print(g);
print(h);