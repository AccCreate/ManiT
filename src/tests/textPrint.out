 _______ _________ _______  _______           _______  _          _______  ______            _______  _______  ______   _______
(  ____ \\__   __/(  ____ \(  ____ )|\     /|(  ____ \( (    /|  (  ____ \(  __  \ |\     /|(  ___  )(  ____ )(  __  \ (  ____ \
| (    \/   ) (   | (    \/| (    )|| )   ( || (    \/|  \  ( |  | (    \/| (  \  )| )   ( || (   ) || (    )|| (  \  )| (    \/
| (_____    | |   | (__    | (____)|| (___) || (__    |   \ | |  | (__    | |   ) || | _ | || (___) || (____)|| |   ) || (_____
(_____  )   | |   |  __)   |  _____)|  ___  ||  __)   | (\ \) |  |  __)   | |   | || |( )| ||  ___  ||     __)| |   | |(_____  )
      ) |   | |   | (      | (      | (   ) || (      | | \   |  | (      | |   ) || || || || (   ) || (\ (   | |   ) |      ) |
/\____) |   | |   | (____/\| )      | )   ( || (____/\| )  \  |  | (____/\| (__/  )| () () || )   ( || ) \ \__| (__/  )/\____) |
\_______)   )_(   (_______/|/       |/     \|(_______/|/    )_)  (_______/(______/ (_______)|/     \||/   \__/(______/ \_______)
